Makefile

```diff
@@ -39,7 +39,7 @@ export GO111MODULE=on
 #
 # Kubebuilder.
 #
-export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.22.0
+export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.23.1
```

Comment:
Q: should we document in https://cluster-api.sigs.k8s.io/reference/versions.html that this is the Kuberenets version we are testing management clusters with, as well as the minimum Kubernetes version we run tests against?

Reply:
Sounds reasonable. I'll take a closer look at the doc tomorrow and update accordingly.

Reply:
Rolled back as discussed.